addons/web_editor/static/lib/odoo-editor/test/spec/list.test.js

```diff
@@ -4290,7 +4281,6 @@ describe('List', () => {
                                 contentBefore: '<ul><li>a</li></ul><ol><li><p>[]b</p></li></ol>',
                                 stepFunction: async editor => {
                                     await deleteBackward(editor);
-                                    await deleteBackward(editor);
                                 },
                                 contentAfter: '<ul><li>a[]b</li></ul>',
```

Comment:
I tried this one in Google Docs and it doesn't do that. It does `<ul><li>a</li></ul><p>[]b</p>`. cc @Zinston

Reply:
Seeing the other behaviors we commented above (the ones I thought were inconsistent with the rest), I would have expected the same as Google Docs indeed because we saw that `<ul><li>[]a</li></ul>` -> backspace -> `<p>[]a</p>`.